### PR TITLE
kola: upload the generated ssh key to aws

### PR DESCRIPTION
**Pull request by crawford**
This fixes the empty Ignition tests.

### CODE REVIEW

**Comment by mischief** [comment]
lgtm assuming the tests pass :-)

File: platform/base.go [hunk 1]
```diff
@@ -123,3 +132,14 @@ func (bc *baseCluster) GetDiscoveryURL(size int) (string, error) {
 	}
 	return string(body), nil
 }
+
+// newName returns a random name prefixed by base.
+func newName(base string) (string, error) {
+	randBytes := make([]byte, 16) //128 bits of entropy
```
Comment (marineam) [inline platform/base.go]: I vote if it looks like a UUID it should actually be a UUID

**Comment by crawford** [comment]
```
$ bin/kola run coreos.ignition.v2.empty --platform=aws --aws-ami=ami-878074e7 --basename crawford
2016-05-13T19:46:33Z kola: === RUN coreos.ignition.v2.empty on aws
2016-05-13T19:47:47Z kola: --- PASS: coreos.ignition.v2.empty on aws (74.313s)
2016-05-13T19:47:47Z kola: 1 passed 0 failed 0 skipped out of 1 total
```

File: platform/base.go (hunk 1 above)
Comment (crawford) [inline platform/base.go]: I just moved that code from the GCE platform. I'm not partial to UUIDs and kinda liked the idea of using a timestamp. Do you have a preference?

Comment (marineam) [inline platform/base.go]: UUIDs contain timestamps ;-)

Comment (marineam) [inline platform/base.go]: whatever is fine with me really

Comment (crawford) [inline platform/base.go]: UUIDs!

File: platform/base.go [hunk 2]
```diff
@@ -34,9 +35,11 @@ type baseCluster struct {
 
 	machlock sync.Mutex
 	machmap  map[string]Machine
+
+	name string
 }
 
-func newBaseCluster() (*baseCluster, error) {
+func newBaseCluster(basename string) (*baseCluster, error) {
 	// set reasonable timeout and keepalive interval
```
Comment (mischief) [inline platform/base.go]: is basename now unused?

Comment (crawford) [inline platform/base.go]: pushed a tad early. check again ;)

Comment (mischief) [inline platform/base.go]: looks good

**Comment by marineam** [comment]
LGTM

**Pull request merged**